.github/workflows/docs.yml

```diff
@@ -0,0 +1,80 @@
+---
+name: Build Documentation
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
```

Comment:
Do we want to update the documentation this often? We could also schedule to do it at night or once a week/month using cron syntax: https://jasonet.co/posts/scheduled-actions/